ethereum.star

```diff
@@ -3,7 +3,7 @@ ethereum_package = import_module(
 )
 
 GETH_IMAGE = "ethereum/client-go:v1.14.0"
-LIGHTHOUSE_IMAGE = "sigp/lighthouse:v5.1.3"
+LIGHTHOUSE_IMAGE = "ethpandaops/lighthouse:stable-minimal"
```

Comment:
The `ethpandaops/ligthouse:stable-minimal` image is rebuilt on every commit, risking package breakage. Hence, we prefer tagged versions for stability. Were any features missing in `sigp/lighthouse` that led to using the alternative image? We could upgrade to the latest stable [v5.2.0](https://github.com/sigp/lighthouse/releases/tag/v5.2.0) release.

Reply:
I changed to `ethpandaops/lighthouse:stable-minimal` because `sigp/lighthouse` didn't support minimal mode. After upgrading it to v5.2.0 things are working fine. Will bump the version to `v5.2.0`.

Reply:
nice!